gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -141,6 +150,94 @@ fluid.defaults("gpii.flowManager.local", {
     }
 });
 
+fluid.defaults("gpii.flowManager.capture", {
```

Comment:
This material is what should go into the separate file Capture.js . The FlowManager already is overfull